File: campus-second-backend/src/main/java/com/campus/entity/Aboutus.java
package com.campus.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.time.LocalDateTime;

/**
 * 关于我们
 */
@Data
@TableName("aboutus")
public class Aboutus {
    @TableId(type = IdType.AUTO)
    private Long id;
    private LocalDateTime addtime;
    private String title;
    private String subtitle;
    private String content;
    private String picture1;
    private String picture2;
    private String picture3;
}

File: campus-second-frontend/src/api/aboutus.ts
import request from '../utils/request'

export interface AboutusItem {
  id: number
  title: string
  subtitle: string
  content: string
  picture1: string
  picture2: string
  picture3: string
}

export const getAboutusDetail = () =>
  request.get<any>('/api/aboutus/detail').then((res: any) => res.data ?? res)

export const updateAboutus = (data: Partial<AboutusItem>) =>
  request.put<any>('/api/aboutus/update', data).then((res: any) => res.data ?? res)
